### ISSUE TITLE
New Applications Still Allow Single Admin

### ISSUE THREAD

**Issue by aomuse** [opencollective/opencollective#5758]
**Describe the bug**
OCF has turned on its the requirement that new collectives require two admins to be hosted, but applications still appear to allow a single admin (ex. [ICDE](https://opencollective.com/icde); [Groomfly](https://opencollective.com/groomflymentoring); and [Resonate](https://opencollective.com/resonate-turtle-island))

**Expected behavior**
Our expectation when this feature was activated was that the application flow would stop the applicant from proceeding when a single admin was present on the project.

**Comment by kaylarep** [comment]
[Slack convo for additional context](https://opencollective.slack.com/archives/GFH4N961L/p1657131071194159?thread_ts=1657048223.068399&cid=GFH4N961L)

**Comment by Betree** [comment]
cc @kewitz 
From what I see, the minimum number of admins is displayed but not enforced in the [create collective form](https://staging.opencollective.com/foundation/apply/form), both for OCF and the general flow.

**Comment by alanna** [comment]
@kewitz is this something you'll fix soon or will it need to be considered for prioritisation in the next sprint? It's fairly urgent for OSC and OCF as the feature isn't working.

**Comment by kaylarep** [comment]
I'm not sure how urgent this is -- it doesn't quite work as intended right now, but that just means our processes (on the OCF side) haven't really changed at all since the release of this feature.

**Comment by alanna** [comment]
True we can manage, but a main purpose of the feature was to address this, so if we can fix it we should.

**Comment by kewitz** [comment]
This is by design. During the speccing and design discussion, there was a divergence in the idea of filtering out these collectives and allowing or not to apply.

I'll make sure to update the rules with respect to being able to apply or not.

**Comment by alanna** [comment]
Applications are still coming into OCF with one admin. This really slows our onboarding process down and defeats a main value of this feature for OCF.

**Comment by alanna** [comment]
@kewitz can you clarify what needs to happen to get this fixed? Do we need to get this prioritised in a sprint as a new feature? Or can it be updated easily? It's continuing to cause problems for OCF. Just had a whole grant application timeline get messed up waiting on them to add a second admin so we can approve them....

**Comment by kewitz** [comment]
Every collective should still expect applications with pending admin invitations to still go through, for example:
![Screen Shot 2022-09-12 at 10 31 14](https://user-images.githubusercontent.com/2119706/189667428-d898f6e3-a118-4b3c-be63-a9fba8a096d8.png)

Why do we let these applications go through?

1. Because every collective that applies through OCF onboarding flow will never have two admins. Therefore enforcing two admins would defeat the purpose of the onboarding flow.
2. Because this flow allows OCF to deal with exceptions by itself.

How to solve this?

1. We can filter out this type of application in the host dashboard and add a new filter option to the dropdown to display applications that have admin invitations pending.
2. We can enforce the policy all the way in the sign-up form, although this would also require us to disable the ability to create new collectives when applying to a host. The user would need to create a collective on the website and after having enough admins use the "Apply to Host" button and pick the existing collective.

**Comment by alanna** [comment]
Thanks @kewitz - I have asked the OCF team to think about how they want this to actually work. I think something like it won't let you through until you *invite* a second admin, but it does let it through with that invitation pending and not yet accepted, could be a good compromise.

**Comment by kewitz** [comment]
> something like it won't let you through until you invite a second admin, but it does let it through with that invitation pending and not yet accepted, could be a good compromise.

This is how it currently works. In the host dashboard there will be an attention sign notifying that the collective still has admin invitations pending:
![](https://user-images.githubusercontent.com/2119706/189667428-d898f6e3-a118-4b3c-be63-a9fba8a096d8.png)

**Comment by natehn** [comment]
What is the reason for allowing Collectives with a pending invite to a second admin to go through? Why not just require two accepted admins?

**Comment by alanna** [comment]
True. It could just sit there until the second one accepts the invitation, then go through to the host. As long as we communicate to people that their application will not progress until the second admin accepts, I think that would be best for OCF. Not sure about other hosts.

**Comment by kaylarep** [comment]
> > something like it won't let you through until you invite a second admin, but it does let it through with that invitation pending and not yet accepted, could be a good compromise.
> 
> This is how it currently works. In the host dashboard there will be an attention sign notifying that the collective still has admin invitations pending: ![](https://user-images.githubusercontent.com/2119706/189667428-d898f6e3-a118-4b3c-be63-a9fba8a096d8.png)

I almost never use this page to view our applications. I probably could, but its not part of my workflow. I can start using it, but its an extra step I haven't needed up to now.

(Not saying we should get rid of that page or anything, its certainly still useful to have all the applications in one place.)

**Comment by natehn** [comment]
Do we currently get an email for these application, with a pending admin? I thought that the motivation was initially from the ticket management side of things

**Comment by kewitz** [comment]
> What is the reason for allowing Collectives with a pending invite to a second admin to go through? Why not just require two accepted admins?

It is written above:

> Why do we let these applications go through?
> 1. Because every collective that applies through OCF onboarding flow will never have two admins. Therefore enforcing two admins would defeat the purpose of the onboarding flow.
> 2. Because this flow allows OCF to deal with exceptions by itself.

@alanna exactly, that's is a valid solution. We can just "hide" these applications until they comply with the policy.

@kaylarep what exactly is the flow you use? Is there any other place you can accept applications where we're missing this alert information?

**Comment by aomuse** [comment]
Would it be possible for us to see pending invitations and initiate push reminders on demand for applications awaiting a second admin? I actually am a bit contrary here. I think the current flow is fine and see all recent suggestions adding a lot more friction to what can be a confusing process for a new applicant.

A process that reminds them at the end that we cannot move forward without the other admin accepting and which allows us to push a reminder would be acceptable for me.

**Comment by kaylarep** [comment]
I don't love the idea of hiding applications till they comply, I worry people will miss it, and then we won't easily be able to ping/notify them.

@kewitz I accept applications directly from their OC url

@aomuse 's ideas are interesting. I do feel that our currently customized onboarding flow is still confusing, as it pushes people thru to the platform without them understanding what they are doing (Joining a fiscal host *&* a web platform). 

+ we call them 'applications' even though its really just their Collective profile (btw SCN also uses 'applications' & 'Collective profile' interchangeably like we do, so their insight into onboarding could be helpful too). But I think you may have better insight than me - do you find that most people in onboarding calls come in confused or understanding?  Also worth considering: is having an onboarding call with each new Collective something we want to do long-term. SCN mentioned this may not be sustainable for them, and they may shift to doing group onboardings + more demo videos.
